### ISSUE TITLE
improve albums/singles tab 

### ISSUE THREAD

**Issue by WHONEYR** [maxrave-dev/SimpMusic#451]
currently the albums and singles tab on simpmusic is too simple, showing just one version of albums and singles and sometimes even unnecessary duplicates.

my suggestion is that they rely a little on the spotify or youtube music tab to show the different versions of an album/single, I think this would be more diverse and simpler, because currently to access an alternative version of an album you have to search a specific song from that album and access it through the song ![Screenshot_20240717_130823_SimpMusic.jpg](https://github.com/user-attachments/assets/5a5e2475-9ba8-4346-8ecf-847ad804b48e)

![Screenshot_20240717_130156_Spotify.jpg](https://github.com/user-attachments/assets/48d91ea4-aab0-4be7-a74a-9b06e3f52956)

![Screenshot_20240717_130813_SimpMusic.jpg](https://github.com/user-attachments/assets/1a9eac9a-bd8b-49a9-8162-165d388905ae)

![Screenshot_20240717_130233_SimpMusic.jpg](https://github.com/user-attachments/assets/a43c7e6b-bb10-4f04-a750-974ada852d8c)

![Screenshot_20240717_130922_YouTube Music.jpg](https://github.com/user-attachments/assets/238a6c1d-e469-48f7-9933-060578b2f07a)

![Screenshot_20240717_130831_SimpMusic.jpg](https://github.com/user-attachments/assets/b9ef547d-f560-4991-9a44-6cf6c8a3a3fa)

**Comment by maxrave-dev** [comment]
btw, SimpMusic data following YouTube Music Web, you can check out:
https://music.youtube.com/browse/MPADUC0076UMUgEng8HORUw_MYHA

**Comment by WHONEYR** [comment]
Yes, and when you go to the album tab on YouTube Music and swipe down and click on the alternative versions of the album, I think this alternative versions section would be a good addition, here is an example (swipe to the end of the tracklist): https://music.youtube.com/playlist?list=OLAK5uy_kQF1D-3BrTCPp38u3eZx5iPgbY2x8wIAg&si=s58u57JCd_805Ina
